### PR TITLE
pronouns: pronoun.is has been dead for months; switch to our own backend

### PR DESCRIPTION
**Pull request by dgw**
### Description
Since @lucasew was kind (and industrious) enough to make a pronoun.is clone written in Svelte, which we can fork (https://github.com/sopel-irc/pronoun-service) and host a copy of on our existing Netlify plan, now seems like the perfect time to replace the old, much cuter, domain name with one that actually works. 😢

### Checklist
- [x] I have read [CONTRIBUTING.md](https://github.com/sopel-irc/sopel/blob/master/CONTRIBUTING.md)
- [x] I can and do license this contribution under the EFLv2
- [x] No issues are reported by `make qa` (runs `make quality` and `make test`)
- [x] I have tested the functionality of the things this change touches

### CODE REVIEW

**Review by SnoopJ — APPROVED** [review]
Straightforward changeset, LGTM

File: sopel/modules/pronouns.py [hunk 1]
```diff
@@ -61,9 +61,15 @@ def setup(bot):
     # and now try to get the current one
     # who needs an API that might never exist?
     # (https://github.com/witch-house/pronoun.is/pull/96)
+    # and is tied to a site that went offline?
+    # (https://github.com/witch-house/pronoun.is/issues/147)
+    # when we can fork someone's Svelte project:
+    # (https://github.com/lucasew/svelte-pronounisland)
+    # and have our own backend that won't go down unless our site does:
+    # (https://pronouns.sopel.chat/)
```
[the remark this answers is not in the document]
Comment (dgw) [reply sopel/modules/pronouns.py]: Fair. Have done.

File: sopel/modules/pronouns.py [hunk 2]
```diff
@@ -152,7 +158,7 @@ def pronouns(bot, trigger):
             # gender, but like… it's a bot.
             bot.say(
                 "I am a bot. Beep boop. My pronouns are it/it/its/its/itself. "
-                "See https://pronoun.is/it for examples."
+                "See https://pronouns.sopel.chat/it for examples."
```
[the remark this answers is not in the document]
Comment (dgw) [reply sopel/modules/pronouns.py]: image

File: sopel/modules/pronouns.py [hunk 3]
```diff
@@ -166,8 +166,15 @@ def say_pronouns(bot, nick, pronouns):
             break
         short = pronouns
 
-    bot.say("{}'s pronouns are {}. See https://pronoun.is/{} for "
-            "examples.".format(nick, pronouns, short))
+    bot.say(
+        "{nick}'s pronouns are {pronouns}. See {BACKEND}/{short} for examples."
```
Comment (SnoopJ) [inline sopel/modules/pronouns.py:170]: Love to see `str.format()` with explicit field names :grin:

**Review by SnoopJ — APPROVED** [review]

**Pull request merged**